infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -445,11 +453,16 @@ exports.tasks.push({
         }
 
         dockerCompose.services[`${name}-${proc}`] = {
-          volumes: devVolumes,
+          volumes: [
+            ...devVolumes,
+            `./services/${name}:/app/services/${name}`, // service should only care about own code
```

Comment:
this is to prevent all services from restarting at the same time when you change something for one service only